css/components/button.css

```diff
@@ -51,7 +47,6 @@
     background-color: hsl(195,83%,68%);
```

Comment:
to maintain consistency, you may want to change it to a variable as well

Reply:
sure i will change

Reply:
done with variable name